docs/usage.md

```diff
@@ -235,6 +236,10 @@ If you prefer, you can use [HISAT2](https://ccb.jhu.edu/software/hisat2/index.sh
 
 To use HISAT2, use the parameter `--aligner hisat2` or set `params.aligner = 'hisat2'` in your config file. Alternatively, you can also use `--aligner salmon` if you want to just perform a fast mapping to the transcriptome with Salmon (you will also have to supply the `--transcriptome` parameter or both a `--fasta` and `--gtf`/`--gff`).
 
+### HISAT2
+
+ To change the `--mp` parameter of HISAT2, we have added 2 params `hisat2_mp_min` and `hisat2_mp_max`. If you specified only one of those, the default value will be used for the other (6 for max, 2 for min). 
```

Comment:
```suggestion
To change the `--mp` parameter of HISAT2, we have added 2 params `hisat2_mp_min` and `hisat2_mp_max`. If you specified only one of those, the default value will be used for the other (6 for max, 2 for min).
```

The error mentioned a trailing empty space, not a leading empty space...
It was at the end of the line :man_facepalming: 
This suggestion should fix it